pom.xml

```diff
@@ -343,11 +333,13 @@
             <configuration>
               <failOnWarning>true</failOnWarning>
               <ignoredDependencies>
+                <ignoredDependency>log4j:log4j:jar:*</ignoredDependency>
```

Comment:
It should be explicitly declared in the test scope if its classes are used in the test scope.